frontend/src/components/regions/UserActions/ActivityDashboardBarGraph.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import {
+  BarChart,
+  Bar,
+  XAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend,
+} from "recharts";
+
+import { eventTypeColors } from "../../../colors";
+
+const RED = eventTypeColors.CARD_MOVED_TO_REVIEW_FEEDBACK;
```

Comment:
What if we decide to make the CARD_MOVED_TO_REVIEW_FEEDBACK color purple instead? 
 I dont see the value in renaming the variables.